notebooks/tfx_pipelines/pipeline/solutions/tfx_pipeline_vertex.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>Nit: consider giving this a vertex_ai alias just so learners understand this is the Vertex Python SDK. I saw that Khalid and the Vertex examples had adopted this approach to provide clarity and had done so as well but want to emphasize I don't feel strongly about this one. Let me know what you think.</p><p> </p><pre class="ql-syntax" spellcheck="false">from google.cloud import aiplatform as vertex_ai
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done!</p>